incubator-calls.md

```diff
@@ -0,0 +1,148 @@
+# TC39 Incubator Call Participation Guidelines
+
+A model of the TC39 working mode is this flowchart:
+
+![TC39 Working Mode](images/tc39-working-mode.svg)
+
+Incubator calls seek to speed up the process by having a sanctioned bi-weekly
```

Comment:
bi-weekly can be confusing. maybe "every two weeks"?

Reply:
+1